### DISCUSSION TITLE
OnePub in GH Actions - team permission missing

### DISCUSSION THREAD

**Discussion by MOPineyro** [onepub-dev/onepub#390] [Support] [answered]
I'm on a free account within the trial period. Within my flutter build workflow, I am able to successfully login using the generated CI token. However, when I run `pub get` I'm still receiving an error.

Here's my step:
```
- name: Run `flutter pub get`
        run: |
          dart pub global activate onepub
          onepub import
          flutter pub get
```

Here's some logs from the action:
```
OnePub version: 3.1.0 

Configure OnePub
Installed OnePub version: 3.1.0.
Importing OnePub secret from ONEPUB_TOKEN
...
Running "flutter pub get" in APP...
Resolving dependencies...
Because APP depends on APP_PACKAGE any which doesn't exist (could not find package APP_PACKAGE at ONEPUB_URL), version solving failed.
pub get failed
```

All of this works seamlessly locally. Am I missing something here?

**Comment by bsutton** [comment]
can you provide a copy of your pubspec.yaml.

The case of your package name 'APP_PACKAGE' looks suspect as package names are normally lower case.

Do you have a macro/environment var called 'APP_PACKAGE' which hasn't expanded correctly?

**Comment by bsutton** [comment]
I've just checked and there is no package called 'APP_PACKAGE' that has been uploaded to onepub, so your package name is incorrect.

**Reply by zacherygentry** [reply]
I'm on a team with @MOPineyro. In his comment, 'APP_PACKAGE' is not our actual package name. We have just redacted the actual name for privacy. As also mentioned in his comment, we are attempting to run these commands in a step on GH actions (with ONEPUB_TOKEN as an env variable), but are getting the `could not find package APP_PACKAGE at ONEPUB_URL` error.
```
flutter pub global activate onepub
onepub import
flutter pub get
```
These same commands work just fine locally, so not sure what the difference is in GH Actions.

**Reply by MOPineyro** [reply]
> I've just checked and there is no package called 'APP_PACKAGE' that has been uploaded to onepub, so your package name is incorrect.

As @zacherygentry mentioned the package name was redacted. The actual package can be found here: https://onepub.dev/packages/betr_fantasy_plugin/spvsdavuhn

**Comment by bsutton** [comment, marked as answer]
OK, apologies, I should have guessed that you had redacted the package.

So the problem is that you have created a team 'betr' and associated the package with that team.

You CI/CD account however is not a member of the' betr' team.

Go into OnePub | Members,
find the CI/CD member
Click the 'Teams' tab
And the click the checkbox next to the 'betr' team.
Click save.
The CI/CD role should now have access to the package.

Any problems let me know.
